frontends/concrete-python/tests/execution/test_others.py

```diff
@@ -1003,6 +1003,14 @@ def issue650(x):
             {},
             id="np.min(x, 0)",
         ),
+        pytest.param(
```

Comment:
why does this new test appear in this PR

Reply:
It's because of the issue I've found. I will split it into two commits.